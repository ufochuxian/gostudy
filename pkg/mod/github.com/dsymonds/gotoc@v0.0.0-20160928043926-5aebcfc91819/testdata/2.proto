// Nested message, plus some message references.

syntax = "proto2";

message A {
  message B {
  }
  optional B b = 1;
}

message C {
  optional A a = 1;
  optional A.B b = 2;
}
