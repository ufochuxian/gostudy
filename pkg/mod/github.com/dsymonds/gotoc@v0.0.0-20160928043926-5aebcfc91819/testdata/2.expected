file: <
  name: "2.proto"
  message_type: <
    name: "A"
    field: <
      name: "b"
      number: 1
      label: LABEL_OPTIONAL
      type: TYPE_MESSAGE
      type_name: ".A.B"
    >
    nested_type: <
      name: "B"
    >
  >
  message_type: <
    name: "C"
    field: <
      name: "a"
      number: 1
      label: LABEL_OPTIONAL
      type: TYPE_MESSAGE
      type_name: ".A"
    >
    field: <
      name: "b"
      number: 2
      label: LABEL_OPTIONAL
      type: TYPE_MESSAGE
      type_name: ".A.B"
    >
  >
>
